Sriram/Reg.html

```diff
@@ -11,9 +11,9 @@ <h2><strong>Register</strong></h2>
 
     <form id="Form">
       First name:
-      <input type="text" name="firstname" value="First name" required/>
+      <input type="text" name="firstname" placeholder="First name" autocomplete="on" required/>
       Last name:
-      <input type="text" name="lastname" value="Last name" /><br /><br />
+      <input type="text" name="lastname" placeholder="Last name" autocomplete="on" /><br /><br />
```

Comment:
Don't use `on`. Use specific ones. Refer this: [The HTML autocomplete attribute - MDN web docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete)

Reply:
@Nuthi-Sriram Last name as well. Not just first.